pom.xml

```diff
@@ -132,6 +132,28 @@
             <artifactId>jsoup</artifactId>
             <version>1.13.1</version>
         </dependency>
+        <!-- 
```

Comment:
Perhaps leave in the comment a text why these dependencies were added in commented out form. Perhaps a URL to where you got this from?

Something like: "We might need this in the future when we add more elaborate testing."

Reply:
I think I will remove them instead. This was added based on Jenkins plugin developer's testing documentation in https://www.jenkins.io/doc/developer/testing/. However, after reading more carefully, these dependencies are needed if we need elaborate testing using pipelines, actual Jenkins instance, Jenkins object model and Web interactions. For now, we we will use mocking for the unit tests and we don't plan to setup the more elaborate testing yet in the foreseable future.

Reply:
Fixed in https://github.com/jenkinsci/tics-plugin/pull/28/commits/f3c90e11a2ef8695866d233c19012e94616fa235